src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -108,6 +108,11 @@ internal void GetDeclarationDiagnostics(DiagnosticBag addTo)
             foreach (var typeParam in _typeParameters)
             {
                 typeParam.ForceComplete(null, default(CancellationToken));
+
+                if (typeParam.HasUnmanagedTypeConstraint)
+                {
+                    addTo.Add(ErrorCode.ERR_UnmanagedConstraintWithLocalFunctions, typeParam.GetNonNullSyntaxNode().Location);
```

Comment:
>addTo.Add(ErrorCode.ERR_UnmanagedConstraintWithLocalFunctions, typeParam.GetNonNullSyntaxNode().Location); [](start = 20, length = 106)

It feels like typeParam.ForceComplete is the right place to report this error.

Reply:
Since we decided to enable local functions, the last parameter would be false. Will add that now.

---
In reply to: [172418267](https://github.com/dotnet/roslyn/pull/24817#discussion_r172418267) [](ancestors = 172418267)

Reply:
Will remove since we decided to enable local functions.

---
In reply to: [172418267](https://github.com/dotnet/roslyn/pull/24817#discussion_r172418267) [](ancestors = 172418267)

Reply:
Belongs in  typeParam.ForceComplete?

---
In reply to: [175998657](https://github.com/dotnet/roslyn/pull/24817#discussion_r175998657) [](ancestors = 175998657,172418267)

Reply:
Consolidated into the other comment.

---
In reply to: [177237879](https://github.com/dotnet/roslyn/pull/24817#discussion_r177237879) [](ancestors = 177237879,175998657,172418267)